src/main/groovy/com/ullink/gradle/opencover/OpenCover.groovy

```diff
@@ -74,7 +74,7 @@ class OpenCover extends ConventionTask {
 
         commandLineArgs += ["-target:${getTargetExec()}", "\"-targetargs:${getTargetExecArgs().collect({escapeArg(it)}).join(' ')}\"", "-targetdir:${project.buildDir}"]
         def filters = getTargetAssemblies().collect { "+[${FilenameUtils.getBaseName(project.file(it).name)}]*" }
-        commandLineArgs += '-filter:"' + filters.join(' ') + '"'
+        commandLineArgs += '-filter:\\"' + filters.join(' ') + '\\"'
```

Comment:
Why would these be needed? For example on line 75 we don't escape them. Same on 76.

Also reminds me of: https://github.com/Ullink/gradle-opencover-plugin/pull/9/files